xml_inscriptions/transcribed/PA.Phil.UP.UM.L.L-1033-51.xml

```diff
@@ -152,14 +155,17 @@
         <body>
             <div type="edition">
             <ab>
-                <lb n="1"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erpernae</name></persName> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>iberti</ex></expan> <gap reason="lost" unit="character" extent="unknown"/>
+                <lb n="1"/> <persName><supplied reason="lost"><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></supplied> <name type="gentilicium" key="Perperna"><supplied reason="lost">P</supplied>erpernae</name></persName> <persName><name type="praenomen" key="Marcus"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <expan><abbr>l</abbr><ex>iberti</ex></expan> <unclear reason="lost">N(?)</unclear> <gap reason="lost" unit="character" extent="unknown"/>
```

Comment:
As a rule don't put text like "(?)" in the text; use attributes and the XSLT will provide those for you. I've made the appropriate change to the file already.